eng/Versions.props

```diff
@@ -60,7 +60,7 @@
     <SystemThreadingAccessControlVersion>10.0.0-alpha.1.25073.13</SystemThreadingAccessControlVersion>
     <VSRedistCommonNetCoreSharedFrameworkx64100PackageVersion>10.0.0-alpha.1.25073.13</VSRedistCommonNetCoreSharedFrameworkx64100PackageVersion>
     <!-- wcf -->
-    <SystemServiceModelVersion>4.10.0</SystemServiceModelVersion>
+    <SystemServiceModelVersion>6.0.0</SystemServiceModelVersion>
```

Comment:
6.0 version of packages are out of support

Reply:
How about 4.10.3 for System.ServiceModel.Duplex and Security and 8.1.1 for the rest? See here - https://github.com/dotnet/windowsdesktop/commit/8cdeb54a8f8928da38399a458d604979c5d807a9

Reply:
Duplex and Security aren't needed any more. The api's from those two packages are now in primitives. Inside the primitives package, in the lib folder, you'll find the assemblies System.ServiceModel.Duplex.dll and System.ServiceModel.Security.dll, which type forward to System.ServiceModel.Primitives.dll. This way if someone compiled a library with an older version of the packages, and then an app upgrades to a newer version, anything referencing types in Duplex or Security will still function correctly.

Reply:
Thanks, @mconnew ! This builds - https://github.com/dotnet/windowsdesktop/tree/bump-up-versions @ericstj , do you mind if I update your PR?

Reply:
Thank you @Tanya-Solyanik - please do.  Thank you @mconnew for the clarification.

Reply:
Actually I just pulled it in - no need to take action @Tanya-Solyanik